google/resource_compute_address.go

```diff
@@ -74,9 +105,30 @@ func resourceComputeAddressCreate(d *schema.ResourceData, meta interface{}) erro
 	}
 
 	// Build the address parameter
-	addr := &compute.Address{Name: d.Get("name").(string)}
-	op, err := config.clientCompute.Addresses.Insert(
-		project, region, addr).Do()
+	v0BetaAddress := &computeBeta.Address{
+		Name:        d.Get("name").(string),
+		AddressType: d.Get("address_type").(string),
```

Comment:
Echoing @rosbo's comment: we should explicitly document (or use `Default` to set) the default value here.

Source: https://github.com/terraform-providers/terraform-provider-google/pull/488/files#r142192254